src/main/java/at/porscheinformatik/sonarqube/licensecheck/npm/PackageJsonDependencyScanner.java

```diff
@@ -47,11 +47,12 @@ public Set<Dependency> scan(SensorContext context)
 
         Set<Dependency> allDependencies = new HashSet<>();
 
+        LOGGER.info("Scanning for NPM dependencies (dir={})", fs.baseDir());
         for (InputFile packageJsonFile : fs.inputFiles(packageJsonPredicate))
         {
             context.markForPublishing(packageJsonFile);
 
-            LOGGER.info("Scanning for NPM dependencies (dir={})", fs.baseDir());
```

Comment:
This old logging info was outputting the same base dir for all package.json files in node_modules/ providing no extra benefit. Furthermore, you don't get to see for which package the scan crashes.